aldryn_faq/menu.py

```diff
@@ -38,24 +38,36 @@ def get_nodes(self, request):
                 categories = categories.filter(appconfig=config)
 
         for category in categories:
-            node = NavigationNode(
-                category.safe_translation_getter(
-                    'name', language_code=language),
-                category.get_absolute_url(language=language),
-                category.pk,
-            )
-            nodes.append(node)
-            for question in category.questions.all():
+            try:
+                url = category.get_absolute_url(language=language)
+            except:
```

Comment:
Are you sure you can't limit the scope a bit? This could swallow so many different kinds of errors.